extensions/fairifier-rdf-extension/deri/grefine/rdf/commands/AddPrefixCommand.java

```diff
@@ -26,26 +26,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 		String name = request.getParameter("name").trim();
         String uri = request.getParameter("uri").trim();
         String projectId = request.getParameter("project");
-        String fetchOption = request.getParameter("fetch");
         try {    
         	getRdfSchema(request).addPrefix(name, uri);
-        	if(fetchOption.equals("web")){
```

Comment:
there are different calls for adding a prefix form web and from file, it seems a bit redundant